examples/volume_renderer.py

```diff
@@ -35,11 +35,24 @@ def build_volume_data(cmdline_args):
             volume = h5.getNode(cmdline_args.node)[:].T
 
     volume_data_kwargs = {'raw_data': volume}
+
+    if cmdline_args.filter:
+        sample_filter = example_sample_filter()
+        volume_data_kwargs['volume_filter'] = sample_filter
+
     if cmdline_args.mask:
         volume_data_kwargs['mask_data'] = example_volume_mask(volume)
 
     return VolumeData(**volume_data_kwargs)
 
+def example_sample_filter():
+    class SampleFilter(VolumeFilter):
+        name = 'Sample'
+
+        def filter(self, raw_data):
+            return np.rot90(raw_data)
+
+    return SampleFilter()
```

Comment:
Same as above, no need to have a factory function, just define the class: it builds itself.